com.sap.cloud.lm.sl.cf.process/src/test/java/com/sap/cloud/lm/sl/cf/process/steps/ValidateDeployParametersStepTest.java

```diff
@@ -141,8 +141,7 @@ private void prepareFileService() throws FileStorageException {
                .thenReturn(createFileEntry(EXISTING_BIGGER_FILE_ID, "extDescriptorFile", 1024 * 1024l + 1));
         Mockito.when(fileService.getFile("space-id", NOT_EXISTING_FILE_ID))
                .thenReturn(null);
-        Mockito.when(fileService.addFile(Mockito.eq("space-id"), Mockito.eq("service-id"), Mockito.any(), Mockito.any(),
```

Comment:
Why is there a need to cast Mockito.any() to File explicitly? Doesn't the type inference do this for us?

Reply:
Because FileService has similar method signature:
addFile(String, String, FileUploadProcessor, InputStream)
addFile(String, String, String, File)

In this case we should cast last argument to File or use anyString() for 3rd parameter